tests/acceptance/features/bootstrap/WebDavLockingContext.php

```diff
@@ -287,7 +288,48 @@ public function unlockItemWithLastPublicLockOfOtherItemUsingWebDavAPI(
 	}
 
 	/**
-	 * @When user :user unlocks file/folder :itemToUnlock with the last created lock of file/folder :itemToUseLockOf of user :lockOwner using the WebDAV API
+	 *
+	 * @param string $user
+	 * @param string $itemToUnlock
+	 *
+	 * @return array
+	 *
+	 * @throws Exception|GuzzleException
+	 */
+	private function countLockOfResources(
+		string $user,
+		string $itemToUnlock
+	): array {
+		$user = $this->featureContext->getActualUsername($user);
+		$baseUrl = $this->featureContext->getBaseUrl();
+		$password = $this->featureContext->getPasswordForUser($user);
+		$body
+			= "<?xml version='1.0' encoding='UTF-8'?>" .
+			"<d:propfind xmlns:d='DAV:'> " .
+			"<d:prop><d:lockdiscovery/></d:prop>" .
+			"</d:propfind>";
+		$response = WebDavHelper::makeDavRequest(
+			$baseUrl,
+			$user,
+			$password,
+			"PROPFIND",
+			$itemToUnlock,
+			null,
+			$this->featureContext->getStepLineRef(),
+			$body,
+			$this->featureContext->getDavPathVersion()
+		);
+		$responseXml = $this->featureContext->getResponseXml($response, __METHOD__);
+		$xmlPart = $responseXml->xpath("//d:response//d:lockdiscovery/d:activelock");
+		if (\is_array($xmlPart)) {
+			return $xmlPart;
+		} else {
+			throw new Exception("Could not find lock of resource $itemToUnlock");
```

Comment:
```suggestion
			throw new Exception("Could not find lock of resource for item $itemToUnlock in the response $responseXml->asXml()");
```
log the response in the error message. `->asXml()` may not be true, please check hai :)